lib/content-services/src/lib/aspect-list/aspect-list-dialog.component.spec.ts

```diff
@@ -113,7 +114,7 @@ describe('AspectListDialogComponent', () => {
             excludedAspects: []
         };
         await TestBed.configureTestingModule({
-            imports: [ContentTestingModule, MatDialogModule],
+            imports: [HttpClientTestingModule, ContentTestingModule, MatDialogModule],
```

Comment:
Add the `HttpClientTestingModule` to the `ContentTestingModule` if it's not yet there, and that will reduce the code duplication